components/EventSection.tsx

```diff
@@ -66,52 +72,78 @@ export function EventSection({
           })}
         </div>
       </div>
-      {!dataFuture && <p>No event data</p>}
-      {view === "Upcoming" && (
-        <div className="flex flex-wrap justify-between gap-4">
-          {dataFuture &&
-            dataFuture.slice(0, 4).map((e: DataProps, i: number) => {
-              return (
-                <div
-                  key={i}
-                  className="bg-gray flex gap-6 overflow-hidden column min-w-[30ch]"
-                >
-                  <CalendarEvent
-                    id={e.id}
-                    date_time={e.date_time}
-                    title={e.title}
-                    url={e.url}
-                    date={e.date}
-                    date_utc={e.date_utc}
-                    date2_utc={e.date2_utc}
-                    date_iso={e.date_iso}
-                    description_long={e.description_long}
-                    contact_info={e.contact_info}
-                    description={e.description}
-                  />
-                </div>
-              )
-            })}
-        </div>
-      )}
-      {view === "Weekly" && (
-        <div className="h-0 min-h-[768px] ">
-          <CalendarWeekly
-            today={today}
-            currentDate={currentDate}
-            events={dataPast.concat(dataFuture)}
-          />
+      <div className="flex flex-wrap justify-evenly gap-6">
+        <div className="flex flex-col justify-center items-start min-w-[20%]">
+          <Card className="flex items-start w-full h-full border-none shadow-none">
+            <CardContent className="py-4">
+              <CCVBars />
```

Comment:
In the CCVBars component, I think it should have a margin bottom. It's currently like right up against the events header

`className="mb-4"`

```jsx
import Image from 'next/image'
import CCVBarsImage from '@/assets/CCVBars.png'

export default function CCVBars() {
  return (
    <Image
      className="mb-4"
      src={CCVBarsImage}
      alt="CCVBars"
      width={100}
      height={50}
    />
  )
}
```

Reply:
Ok. I added this default and the ability to override the default with a custom className. I adjusted margins where CCVBars were used elsewhere.